setup.py

```diff
@@ -97,8 +97,13 @@ def package_files(directory):
     install_requires=[],
     python_requires='>=3.5',
     packages=find_packages(),
-    include_package_data=True,
-    package_data={'cocotb': package_files('cocotb/share')},
+    package_data={
+        'cocotb': [
+            *package_files('cocotb/share/makefiles'),
+            *package_files('cocotb/share/include'),
+            *package_files('cocotb/share/def'),
+        ]
```

Comment:
This is still pretty weird compared to
```suggestion
        'cocotb': (
            package_files('cocotb/share/makefiles') +
            package_files('cocotb/share/include') +
            package_files('cocotb/share/def')
        )
```

Reply:
The first allows you to specify particular files because it is already a list. But I'll change it